src/locale/sv-SE.h

```diff
@@ -0,0 +1,189 @@
+#define D_STR_8C_HEAT "8C " D_STR_HEAT
+#define D_STR_10C_HEAT "10C " D_STR_HEAT
+#define D_STR_BUTTON "Knapp"
+#define D_STR_NIGHT "Natt"
+#define D_STR_SILENT "Tyst"
+#define D_STR_FILTER "Filter"
+#define D_STR_3D "3D"
+#define D_STR_CELSIUS "Celsius"
+#define D_STR_FAHRENHEIT "Fahrenheit"
+#define D_STR_CELSIUS_FAHRENHEIT D_STR_CELSIUS "/" D_STR_FAHRENHEIT
+#define D_STR_UP "Upp"
+#define D_STR_TEMPUP D_STR_TEMP " upp"
+#define D_STR_DOWN "Ner"
+#define D_STR_TEMPDOWN D_STR_TEMP " ner"
+#define D_STR_CHANGE "Ändra"
+#define D_STR_START "Starta"
+#define D_STR_STOP "Stoppa"
+#define D_STR_MOVE "Flytta"
+#define D_STR_SET "Ställ in"
+#define D_STR_CANCEL "Avbryt"
+#define D_STR_COMFORT "Komfort"
+#define D_STR_SENSOR "Sensor"
+#define D_STR_DISPLAY "Display"
+#define D_STR_WEEKLY "Veckovis"
+#define D_STR_WEEKLYTIMER D_STR_WEEKLY " timer"
+#define D_STR_WIFI "WiFi"
+#define D_STR_LAST "Senast"
+#define D_STR_FAST "Snabb"
+#define D_STR_SLOW "Sakta"
+#define D_STR_AIRFLOW "Luftflöde"
+#define D_STR_STEP "Steppa"
+#define D_STR_NA "N/A"
+#define D_STR_INSIDE "Inne"
+#define D_STR_OUTSIDE "Ute"
+#define D_STR_LOUD "Hög"
+#define D_STR_UPPER "Övre"
+#define D_STR_LOWER "Nedre"
+#define D_STR_BREEZE "Bris"
+#define D_STR_CIRCULATE "Cirkulera"
+#define D_STR_CEILING "Tak"
+#define D_STR_WALL "Vägg"
+#define D_STR_ROOM "Rum"
+#define D_STR_6THSENSE "6e sinne"
+#define D_STR_ZONEFOLLOW "Följ zon"
+#define D_STR_FIXED "Fast"
+#define D_STR_TYPE "Typ"
+#define D_STR_SPECIAL "Speciell"
+#define D_STR_RECYCLE "Återvinn"
+#define D_STR_ID "Id"
+#define D_STR_VANE "Vindflöjel"
+
+#define D_STR_AUTO "Auto"
+#define D_STR_AUTOMATIC "Automatisk"
+#define D_STR_MANUAL "Manuell"
+#define D_STR_COOL "Kyla"
+#define D_STR_HEAT "Värme"
+#define D_STR_FAN "Fläkt"
+#define D_STR_FANONLY "fläkt-enbart"
+#define D_STR_FAN_ONLY "fläkt_enbart"
+#define D_STR_ONLY "Enbart"
+#define D_STR_FANSPACEONLY D_STR_FAN " " D_STR_ONLY
+#define D_STR_FANONLYNOSPACE D_STR_FAN D_STR_ONLY
+#define D_STR_DRY "Torka"
+
+#define D_STR_MAX "Max"
+#define D_STR_MAXIMUM "Maximum"
+#define D_STR_MIN "Min"
+#define D_STR_MINIMUM "Minimum"
+#define D_STR_MED "Med"
+#define D_STR_MEDIUM "Medium"
+
+#define D_STR_HIGHEST "Högsta"
+#define D_STR_HIGH "Hög"
+#define D_STR_HI "Hög"
+#define D_STR_MID "Mellan"
+#define D_STR_MIDDLE "Mellan"
+#define D_STR_LOW "Låg"
+#define D_STR_LO "Låg"
+#define D_STR_LOWEST "Lägsta"
+#define D_STR_RIGHT "Höger"
+#define D_STR_MAXRIGHT D_STR_MAX " höger" D_STR_RIGHT
+#define D_STR_RIGHTMAX_NOSPACE D_STR_MAX D_STR_RIGHT
+#define D_STR_LEFT "Vänster"
+#define D_STR_MAXLEFT D_STR_MAX " vänster"
+#define D_STR_LEFTMAX_NOSPACE D_STR_MAX D_STR_LEFT
+#define D_STR_WIDE "Vid"
+#define D_STR_CENTRE "Mitten"
+#define D_STR_TOP "Topp"
+#define D_STR_BOTTOM "Botten"
+
+#define D_STR_ECONOTOGGLE "Toggla eko"
+#define D_STR_EYEAUTO "Auto öga"
+#define D_STR_LIGHTTOGGLE "Toggla svag"
+#define D_STR_OUTSIDEQUIET "Tyst ute"
+#define D_STR_POWERTOGGLE "Toggla strömläge"
```

Comment:
If possible maybe reuse already defined strings ?

Reply:
True, just wanted to make sure.

Reply:
Changed the ones that makes a correct language in regards of when uppercase letter should be present and not.